lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -21,7 +21,7 @@ module CldrRegexpCollection
       CH: /\A\d{4}\z/,
       AT: /\A\d{4}\z/,
       ES: /\A\d{5}\z/,
-      NL: /\A[1-9]\d{3}[ ]?(S[BCE-RT-Z]|[A-RT-Z][A-Z])\z/,
+      NL: /\A[1-9]\d{3}[ ]?(S[BCE-RT-Z]|[A-RT-Z][A-Z])\z/i,
```

Comment:
As an example, Postal codes in NL require uppercase, so this change would need to be reverted: https://en.wikipedia.org/wiki/Postal_codes_in_the_Netherlands